### DISCUSSION TITLE
2s agcn with 2d key points

### DISCUSSION THREAD

**Discussion by PoupayNinja** [open-mmlab/mmaction2#2338] [Q&A] [answered]
Hello,

Is it possible to train 2s agcn model with 2d key points instead of 3d?

Thanks.

**Comment by Dai-Wenxun** [comment, marked as answer]
Our new version 1.0 support it already. https://mmaction2.readthedocs.io/en/latest/get_started/installation.html. https://github.com/open-mmlab/mmaction2/tree/main/configs/skeleton/2s-agcn

**Reply by PoupayNinja** [reply]
Thanks a lot

**Comment by PoupayNinja** [comment]
Do you have a script to generate the pickle annotation file for 2d keypoints as that of https://github.com/open-mmlab/mmaction2/blob/main/tools/data/skeleton/gen_ntu_rgbd_raw.py (I suppose this one is only for 3d keypoints)? Thanks in advance.
